dbt/scripts/utils/export.py

```diff
@@ -0,0 +1,230 @@
+# Shared utilities for exporting models
+import contextlib
+import io
+import json
+import os
+import pathlib
+import shutil
+
+import pandas as pd
+import pyathena
+from dbt.cli.main import dbtRunner
+from openpyxl.styles import Alignment
+from openpyxl.utils import column_index_from_string, get_column_letter
+from openpyxl.worksheet.table import Table, TableStyleInfo
+
+# Shared object for running dbt CLI commands
+DBT = dbtRunner()
+
+
+def export_models(
+    target: str = "dev",
+    select: list[str] | None = None,
+    selector: str | None = None,
+    rebuild: bool = False,
+    where: str | None = None,
+):
+    """
+    Export a group of models to Excel workbooks in the output directory
+    `export/output/`.
+
+    Arguments:
+
+        * target (str): dbt target to use for querying model data, defaults to
+            "dev"
+        * select (list[str]): One or more dbt --select statements to
+            use for filtering models
+        * selector (str): A selector name to use for filtering
+            models, as defined in selectors.yml. One of `select` or `selector`
+            must be set, but they can't both be set
+        * rebuild (bool): Rebuild models before exporting, defaults to False
+        * where (str): Optional SQL expression representing a WHERE clause to
+            filter models
+    """
```

Comment:
This is the only part of the code that changed during the move -- I added a docstring since the code is no longer colocated with the argparse arguments that were previously providing documentation.